### DISCUSSION TITLE
Themes!

### DISCUSSION THREAD

**Discussion by iznaut** [NEW-CYLANDIA/little-warioware#7] [Ideas]
between microgame challenges, there's an "intermission" screen that displays the player's current lives and score for a moment before transitioning to the next microgame

these screens are often styled around whichever character "made" the playlist of games, so they can vary in aesthetics - i think of the elevator or stereo as the "canonical" intermission, but it would be neat if LW supported user-submitted themes!

at it's most basic, this would just be a few images to skin the life icons and background (likely the "Speed/Level Up" and "Game Over" messages as well) so they would need to match dimensions and everything, but maybe in the future we could support fully custom scenes with different icon placement and transitions

i know Godot has some sort of "theme" functionality for UI stuff - i'm not really sure what that's capable of, but maybe it could be helpful for this purpose?

![WarioWare, Inc _ Mega Party Game$! - Full Stage Clear Walkthrough (All Characters) 0-10 screenshot](https://user-images.githubusercontent.com/36773942/208497337-ac559419-653b-4f41-980e-495a904dd6ed.png)
![Longplay of WarioWare, Inc _ Mega Microgame$_Minigame Mania 3-45 screenshot](https://user-images.githubusercontent.com/36773942/208497158-c6b8b961-7d9a-4e77-a057-14ae70336367.png)
![Longplay of WarioWare, Inc _ Mega Microgame$_Minigame Mania 6-47 screenshot](https://user-images.githubusercontent.com/36773942/208497166-07b12b32-962a-4216-b2cd-9f0594517706.png)
![Longplay of WarioWare, Inc _ Mega Microgame$_Minigame Mania 22-28 screenshot](https://user-images.githubusercontent.com/36773942/208497171-6fa60f70-6066-42d5-af7e-c364cfde06cc.png)

**Comment by DrSlowpokePhd** [comment]
So I doubt that Godot UI themes are gonna be helpful here, since they almost exclusively focus on UI elements that get re-used a lot (Buttons, Checkboxes, Labels, panels, etc) but I do think that some sort of definition for user-submitted themes that is custom for Little Warioware would be useful here. Just throw some assets together in a zip with a json file and write an interpreter to interpret all the data.
